pkg/server/smtp/handler.go

```diff
@@ -455,9 +476,12 @@ func (s *Session) readByteLine() ([]byte, error) {
 	if err := s.conn.SetReadDeadline(s.nextDeadline()); err != nil {
 		return nil, err
 	}
-	b, err := s.reader.ReadBytes('\n')
-	if err == nil && s.debug {
-		fmt.Printf("%04d   %s\n", s.id, bytes.TrimRight(b, "\r\n"))
+	b, err := s.text.ReadDotBytes()
+	if err != nil {
+		return nil, err
+	}
+	if s.debug {
+		s.logger.Debug().Msgf("%s", b)
```

Comment:
This log statement is not correct, the previous debug was intentionally not outputting to logs.  Please revert to previous Printf+format string to match the rest present in this file, see `inbucket -help`:
```
  -netdebug
        Dump SMTP & POP3 network traffic to stdout.
```

Reply:
Perhaps it would make sense to just output the number of bytes read for this.  I don't think it's worth the effort to split the incoming message into lines and print each one.  Please do keep the formatting so it's obvious which session the bytes were read for.

Reply:
Example:
```bash
2018-05-06T17:48:46Z |INFO| Inbucket starting buildDate=undefined phase=startup version=undefined
2018-05-06T17:48:46Z |INFO| Retention configured for 24h0m0s module=storage phase=startup
2018-05-06T17:48:46Z |INFO| Web UI content mapped module=web path=ui phase=startup
2018-05-06T17:48:46Z |INFO| Generating random cookie.auth.key module=web phase=startup
2018-05-06T17:48:46Z |DEBU| STARTTLS feature available module=smtp phase=tls
2018-05-06T17:48:46Z |INFO| SMTP listening on tcp4 addr=0.0.0.0:2500 module=smtp phase=startup
2018-05-06T17:48:46Z |DEBU| Retention scanner sleeping for 59.999999666s module=storage
2018-05-06T17:48:46Z |INFO| HTTP listening on tcp4 addr=0.0.0.0:9000 module=web phase=startup
2018-05-06T17:48:46Z |INFO| POP3 listening on tcp4 addr=0.0.0.0:1100 module=pop3 phase=startup
2018-05-06T17:48:50Z |INFO| Starting SMTP session module=smtp remote=127.0.0.1:40538 session=1
0001 > 220 inbucket Inbucket SMTP ready
0001   EHLO ubuntu-s-1vcpu-1gb-nyc3-01
0001 > 250-Great, let's get this show on the road
0001 > 250-8BITMIME
0001 > 250-STARTTLS
0001 > 250 SIZE 10240000
2018-05-06T17:48:50Z |DEBU| Entering state READY module=smtp remote=127.0.0.1:40538 session=1
0001   STARTTLS
2018-05-06T17:48:50Z |DEBU| Initiating TLS context. module=smtp remote=127.0.0.1:40538 session=1
0001 > 220 STARTTLS
2018-05-06T17:48:50Z |DEBU| Entering state GREET module=smtp remote=127.0.0.1:40538 session=1
0001   EHLO ubuntu-s-1vcpu-1gb-nyc3-01
0001 > 250-Great, let's get this show on the road
0001 > 250-8BITMIME
0001 > 250 SIZE 10240000
2018-05-06T17:48:50Z |DEBU| Entering state READY module=smtp remote=127.0.0.1:40538 session=1
0001   MAIL FROM:<me@example.com>
2018-05-06T17:48:50Z |INFO| Mail from: me@example.com module=smtp remote=127.0.0.1:40538 session=1
0001 > 250 Roger, accepting mail from <me@example.com>
2018-05-06T17:48:50Z |DEBU| Entering state MAIL module=smtp remote=127.0.0.1:40538 session=1
0001   RCPT TO:<user@example.com>
2018-05-06T17:48:50Z |DEBU| Recipient added module=smtp remote=127.0.0.1:40538 session=1 to=user@example.com
0001 > 250 I'll make sure <user@example.com> gets this
0001   DATA
2018-05-06T17:48:50Z |DEBU| Entering state DATA module=smtp remote=127.0.0.1:40538 session=1
0001 > 354 Start mail input; end with <CRLF>.<CRLF>
0001   Received 4157 bytes
2018-05-06T17:48:50Z |DEBU| Delivering message mailbox=user module=message
0001 > 250 Mail accepted for delivery
2018-05-06T17:48:50Z |INFO| Message size 4157 bytes module=smtp remote=127.0.0.1:40538 session=1
2018-05-06T17:48:50Z |DEBU| Entering state READY module=smtp remote=127.0.0.1:40538 session=1
0001   QUIT
0001 > 221 Goodnight and good luck
2018-05-06T17:48:50Z |DEBU| Entering state QUIT module=smtp remote=127.0.0.1:40538 session=1
2018-05-06T17:48:50Z |INFO| Closing connection module=smtp remote=127.0.0.1:40538 session=1
2018-05-06T17:48:50Z |WARN| Closing connection error="close tcp4 127.0.0.1:2500->127.0.0.1:40538: use of closed network connection" module=smtp remote=127.0.0.1:40538 session=1
```